rune/libcontainer/configs/enclave.go

```diff
@@ -7,7 +7,11 @@ const (
 )
 
 type Enclave struct {
-	Type string `json:"type"`
-	Path string `json:"path"`
-	Args string `json:"args,omitempty"`
+	Type              string `json:"type"`
+	Path              string `json:"path"`
+	Args              string `json:"args,omitempty"`
+	RaType            string `json:"ra_type,omitempty"`
+	RaSpid            string `json:"ra_spid,omitempty"`
```

Comment:
The below 3 parameters are dedicated for EPID-based RA. So name these variables as RaEpidSpid, RaEpidSubscriptionKey and RaEpidQuoteType.